src/features/ingredient-category-card/ingredient-category-card.contract.ts

```diff
@@ -0,0 +1,8 @@
+import { z } from 'zod';
+
+export const IngredientCategoryCardSchema = z.object({
+  ingredientImageSrc: z.string().url(),
+  ingredientIcon: z.custom<React.ReactNode>().optional(),
+  ingredientTitle: z.string().min(1),
+  ingredientDescription: z.string(),
+});
```

Comment:
IngredientCategoryCard에서 Ingredient Category라는 것이 이름에서 명시되고 있기에 해당 카드의 각 필드에 ingredient를 명시할 이유가 없는것 같습니다.
아래와 같이 수정하시는게 어떨까요?
```suggestion
export const IngredientCategoryCardSchema = z.object({
  imageSrc: z.string().url(),
  icon: z.custom<React.ReactNode>().optional(),
  title: z.string().min(1),
  description: z.string(),
});
```